examples/dreambooth/train_dreambooth_lora.py

```diff
@@ -793,7 +796,7 @@ def main(args):
         cur_class_images = len(list(class_images_dir.iterdir()))
 
         if cur_class_images < args.num_class_images:
-            torch_dtype = torch.float16 if accelerator.device.type == "cuda" else torch.float32
+            torch_dtype = torch.float16 if accelerator.device.type in ("cuda", "xpu") else torch.float32
```

Comment:
We cannot train in mixed-precision?

Reply:
I wasn't aware of it, just want to keep the XPU device has the same precision as CUDA.